endpoint.cpp

```diff
@@ -230,7 +230,7 @@ int Endpoint::read_msg(struct buffer *pbuf, int *target_sysid)
     *target_sysid = -1;
 
     if (msg_entry == nullptr)
-        log_error("No message entry for %u", msg_id);
+        log_debug("No message entry for %u", msg_id);
```

Comment:
we also need to fix this unbalanced use of braces on if and else.  And update the coding style script to catch it.